adapters/rubicon/rubicon.go

```diff
@@ -16,6 +16,7 @@ import (
 
 	"github.com/mxmCherry/openrtb"
 	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/endpoints/info"
```

Comment:
This is a bad dependency... adapters shouldn't depend on endpoints.

For example, it's easy to imagine a CLI tool which can be used to test adapters, but has nothing to do with the HTTP endpoints at all. I could also imagine an Endpoint which lets you call a specific adapter, which would create a circular dependency here.

A better way would be to pull the "info file reading" code into a different module, which both `adapters` and `endpoints` could depend on. Perhaps something like `adapter_metadata`?